senza/cli.py

```diff
@@ -417,11 +417,17 @@ def create(definition, region, version, parameter, disable_rollback, dry_run, fo
     for name, parameter in data.get("Parameters", {}).items():
         parameters.append([name, getattr(args, name, None)])
 
-    tags = {
+    tags = {}
+    for tag in input["SenzaInfo"].get('Tags', []):
```

Comment:
I would make "Tags" a dict (tags have no sorting), i.e.

    for key, val in input["SenzaInfo"].get('Tags', {}).items():
        ..